app/assets/javascripts/type_station/editables/file.js.coffee

```diff
@@ -1,8 +1,16 @@
 buildUploader = (element, data)->
   elementId = $(element).attr('id')
   $input = $('<input/>')
+  $label = $('<label/>')
+
+  $labelIcon = $('<i/>').addClass 'ion-ios-upload ts-file-style-icon'
+  $labelText = $('<span/>').addClass('ts-file-style-text').text('Upload file')
+
+  $label
+    .addClass 'ts-file-style'
+
   $input
-    .attr({type: "file", name: "file", class: 'ts-editable-file-input', 'data-form-data': JSON.stringify(data), 'data-element-id': elementId})
+    .attr({type: "file", name: "file", class: 'ts-editable-file-input', style: 'display: none','data-form-data': JSON.stringify(data), 'data-element-id': elementId})
```

Comment:
`style: 'display: none` - Should probably move this into the ts-editable-file-input class (so it can be changed by the user)